resources/images/s3/filestorage.go

```diff
@@ -46,13 +46,14 @@ const (
 // Data layer for file storage.
 // Implements model.FileStorage interface
 type SimpleStorageService struct {
-	client *s3.S3
-	bucket string
+	client         *s3.S3
+	bucket         string
+	setTenantToken bool
 }
 
 // NewSimpleStorageServiceStatic create new S3 client model.
 // AWS authentication keys are automatically reloaded from env variables.
-func NewSimpleStorageServiceStatic(bucket, key, secret, region, token, uri string) (*SimpleStorageService, error) {
+func NewSimpleStorageServiceStatic(bucket, key, secret, region, token, uri string, setTokenTag bool) (*SimpleStorageService, error) {
```

Comment:
`setTokenTag` is yet another variation of `setTenantToken` and `SettingSetTenantTag`, unify to `set tenant tag`